include/session/sodium_array.hpp

```diff
@@ -0,0 +1,231 @@
+#pragma once
```

Comment:
I'm a bit confused -- didn't we already have this header in here somewhere?

Reply:
Oh never mind, I see this was just moved away from util.hpp.

Reply:
Yea - I think we found this caused some linking issues in the `libQuic` branch - maybe something to do with the `ENABLE_ONIONREQ` flag? Or just trying to avoid a future conflict? I can't remember exactly 😞

They _should_ have also been removed from the `util.cpp` though which they hadn't been so I've cleaned that up